packages/channels/src/slack/README.md

```diff
@@ -0,0 +1,29 @@
+### Sending
+
+| Channels | Slack |
+| -------- | :---: |
+| Text     |  ✅   |
+| Image    |  ✅   |
+| Choice   |  ✅   |
+| Dropdown |  ✅   |
+| Card     |  ✅   |
+| Carousel |  ✅   |
+| File     |  ✅   |
+| Audio    |  ✅   |
+| Video    |  ✅   |
```

Comment:
These work by have markdown with a file url. There's a way to upload files to slack but I couldn't figure out how it work.

Reply:
Maybe you could use the ⚠️ emoji instead of the check, in this case?

Reply:
Nah it's still useable. I would say this would be for stuff where you use actual functionality.